src/Microsoft.PowerApps.TestEngine/SingleTestRunner.cs

```diff
@@ -201,11 +201,11 @@ public async Task RunTestAsync(string testRunId, string testRunDirectory, TestSu
 
                 if (allTestsSkipped)
                 {
-                    // Run test case one by one, mark it as skipped
+                    // Run test case one by one, mark it as failed
                     foreach (var testCase in _testState.GetTestSuiteDefinition().TestCases)
                     {
                         var testId = _testReporter.CreateTest(testRunId, testSuiteId, $"{testCase.TestCaseName}", "TODO");
```

Comment:
(not in your change) What's up with the "TODO" here? Should that be something?

Reply:
looks like it assigns that value as testLocation (Location of test file) in Testreporter.cs while creating a test. Not sure of its significance.